src/stores/salem-store.tsx

```diff
@@ -0,0 +1,100 @@
+import { create } from 'zustand';
+import { persist } from 'zustand/middleware';
+import { Player } from '../types';
+
+type SalemState = {
+    instructionSpeech: boolean;
+    setInstructionSpeech: (enabled: boolean) => void;
+	players: Player[];
+    setPlayerName: (playerId: number, name: string) => void;
+    resetPlayers: () => void;
+	addPlayer: () => void;
+	removePlayer: (playerId: number) => void;
+	movePlayerDown: (playerId: number) => void;
+	movePlayerUp: (playerId: number) => void;
+}
```

Comment:
Got a weird mix of spaces and tabs here. Would be nice if uniform. But we can also introduce a lint rule for this so it doesn't happen again